knowledge-base/actions/nrwl/nx-set-shas/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: "Nx set SHAs"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
```

Comment:
@sph4674 please add `https://github.com/nrwl/nx-set-shas#permissions-in-v2` in reference links.